js/controller.js

```diff
@@ -1410,24 +1410,45 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       if(this.state.isFullScreenSupported) {
         Fullscreen.toggle(this.state.mainVideoInnerWrapper.get(0));
       }
-      // partial support, video element only (iOS)
-      else if (this.state.isVideoFullScreenSupported) {
-        if (this.state.fullscreen) {
-          this.state.mainVideoElement.webkitExitFullscreen();
-        } else {
-          this.state.mainVideoElement.webkitEnterFullscreen();
-        }
+      // partial support, video element only (iOS) and not vr
```

Comment:
Should we have any is iOS checks for any of these iOS only situations?

Reply:
I think that this is not necessary. This code based on the properties of the system. It worked earlier, before this PR. In this PR, we added check for video360, because standard fullscreen does not work correctly with video360.